tracer/test/Datadog.Trace.Tests/FeatureFlags/FeatureFlagsEvaluatorTests.Bundle.cs

```diff
@@ -0,0 +1,195 @@
+        {
+            _ = 0;
+        }
+
+        AssertEqual(testCase.Result.Value, result.Value);
+        AssertEqual(testCase.Result.Variant, result.Variant);
+
+        Assert.NotNull(description);
+
+        void AssertEqual(object? expected, object? obj)
+        {
+            if (type == Trace.FeatureFlags.ValueType.INTEGER && obj is int intObj)
+            {
+                Assert.Equal(Convert.ToInt32(expected), intObj);
+            }
+            else if (type == Trace.FeatureFlags.ValueType.JSON)
+            {
+                if (obj is string jsonTxt)
+                {
+                    Assert.Equal<object>(expected?.ToString(), obj);
+                }
+                else
+                {
+                    Assert.Equal<object>(expected, obj);
+                }
+            }
+            else
+            {
+                Assert.Equal<object>(expected, obj);
+            }
+        }
+    }
+
+    private static Trace.FeatureFlags.ValueType GetVariationType(string? variationType)
+    {
+        return variationType switch
+        {
+            "INTEGER" => Trace.FeatureFlags.ValueType.INTEGER,
+            "NUMERIC" => Trace.FeatureFlags.ValueType.NUMERIC,
+            "STRING" => Trace.FeatureFlags.ValueType.STRING,
+            "BOOLEAN" => Trace.FeatureFlags.ValueType.BOOLEAN,
+            "JSON" => Trace.FeatureFlags.ValueType.JSON,
+            _ => throw new NotImplementedException(),
+        };
+    }
+
+    private static ServerConfiguration ReadConfig()
+    {
+        // Read config
+        var configContent = ResourceHelper.ReadAllText<FeatureFlagsEvaluatorTests>("resources.config.flags-v1.json");
+        var fullObject = JObject.Parse(configContent);
+        var dataToken = fullObject.SelectToken("data.attributes.flags");
+        var flags = dataToken?.ToObject<Dictionary<string, Flag>>();
+        Assert.NotNull(flags);
+
+        foreach (var flag in flags)
+        {
+            if (flag.Value.Allocations is null) { continue; }
+            foreach (var allocation in flag.Value.Allocations)
+            {
+                if (allocation.Rules is null) { continue; }
+                foreach (var rule in allocation.Rules)
+                {
+                    if (rule.Conditions is null) { continue; }
+                    foreach (var condition in rule.Conditions)
+                    {
+                        if (condition.Value is not JArray jArray) { continue; }
+                        var arr = jArray.ToObject<object[]>();
+                        condition.Value = arr;
+                    }
+                }
+            }
+        }
+
+        var config = new ServerConfiguration { Flags = flags };
+
+        return config;
+    }
+
+    private static List<object[]> GetTestData()
+    {
+        // This file should regularly be updated from here https://github.com/DataDog/experimental/blob/main/teams/asm/iast/redaction/suite/evidence-redaction-suite.yml
```

Comment:
If it needs to be, then we should probably set up some automation, because it _won't_ be otherwise 😉